### DISCUSSION TITLE
How to achieve this kind of animation with Remotion

### DISCUSSION THREAD

**Discussion by HamiguaLu** [remotion-dev/remotion#3205] [Q&A] [answered]
![4](https://github.com/remotion-dev/remotion/assets/12206531/10fdede4-15d8-416c-a38e-1a0c2140aa87)

First of all, thanks very much for such a amzaing project!
I want to achieve the above animation with remotion,  there are two images overlapped,  while auto slide, top image will be partialy hidden, finally the bottom image will show up completly

Thanks in advance

**Comment by JonnyBurger** [comment, marked as answer]
It is a wipe transition: https://www.remotion.dev/docs/transitions/presentations/wipe

You can use our transitions package, or if you need to customize it with a line in the middle, refer to the source code of the wipe transition! (Linked at the bottom of the page.)

**Comment by HamiguaLu** [comment]
@JonnyBurger Thanks for the fast reply, I will try later on
Have a nice day
